### PR TITLE
Experiment

### PR DESCRIPTION
**Pull request by OmarMWarraich**
I have made the hamburger icon interactive.
The hamburger icon turns into a X sign when clicked and the menu items appear.
Clicking again on X sign or any of the menu items closes the menu and hamburger icon reappears in place of X.
Clicking on menu items connects the user to the relevant part of the ui.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,21 +8,39 @@
...
-        </section>
+
+
+                    
+            <nav class="nav" id="nav">
+                <div class="logo_header">  
+                  <a href="#" class="logo">Ova</a>
+                  <div class="navbar_toggle">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
```
Comment (tufoinnkuo10) [inline index.html:18]: - [x] Great job so far with the menu. kindly ensure that when the menu comes up the user should not be able to scroll through your portfolio unless the menu is closed or a link is clicked. the menu should serve as a means of navigation when it comes up. great work 😄

File: index.html [hunk 2]
```diff
@@ -8,21 +8,39 @@
...
+                  <li>
+                    <a href="#work-flex" class="nav_link1">Portfolio</a>
+                  </li>
+                  <li>
+                    <a href="#hero" class="nav_link2">About</a>
+                  </li>
+                  <li>
+                    <a href="#form" class="nav_link3">Contact</a>
+                  </li>
+                  <li>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] kindly align your links so they follow the figma design. your close icon should be well positioned also. please make fix that.
your version
Screenshot (271)_LI
figma version
Screenshot (270)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @OmarMWarraich 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (OmarMWarraich) [reply index.html:18]: @tufoinnkuo10  Updated accordingly. The menu really serves as a means of navigation through the ui.

File: index.html (hunk 2 above)
Comment (OmarMWarraich) [reply index.html]: @tufoinnkuo10 I have aligned the elements according to the figma template.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @OmarMWarraich , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**